### PR TITLE
fix composer json error

### PR DESCRIPTION
**Pull request by pythias**

### CODE REVIEW

**Comment by pythias** [comment]
I signed it!

File: examples/php/composer.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "name": "grpc/grpc-demo",
   "description": "gRPC example for PHP",
   "require": {
-    "grpc/grpc": "v1.0.0",
+    "grpc/grpc": "v1.0.0"
```
Comment (thinkerou) [inline examples/php/composer.json]: Why not need comma?

Comment (fr05t1k) [inline examples/php/composer.json]: @thinkerou it's syntax error for last property of object and last item of array in json file.

Comment (thinkerou) [inline examples/php/composer.json]: Oh, got it.

**Comment by stanley-cheung** [comment]
Dupe of #7850

**Pull request closed without merging**